draft-ietf-moq-transport.md

```diff
@@ -1522,6 +1554,11 @@ session with a Protocol Violation ({{session-termination}}).
 
 * Subscribe Parameters: The parameters are defined in {{version-specific-params}}.
 
+PUBLISHER_CHOSEN_SUBSCRIPTION_ID: The Publisher-Chosen Subscription ID for
```

Comment:
What field is this referencing and how would it be omitted?

Reply:
It's intended to be a parameter, so it can be easily optional.  I don't love how we have parameter defs miles away from where they are used.

Reply:
Ok, I now understand what you're trying to do, but given right above it says all parameters are defined elsewhere, this was confusing for me.  Most parameters are shared among messages, and this one isn't, which might be an argument for it being a field?  Other control messages can reference this ID, but can't change it.